bindgen/codegen/mod.rs

```diff
@@ -3798,6 +3830,8 @@ impl CodeGenerator for Enum {
         // do).
         let mut constified_variants = VecDeque::new();
 
+        let mut enum_variants = vec![];
```

Comment:
Seems like this vector should be part of the builder? Then `with_variant` can just append to it and you don't need the outparam.

Reply:
Good point!